src/components/Squeak/hooks/useQuestion.tsx

```diff
@@ -170,6 +172,79 @@ export const useQuestion = (id: number | string, options?: UseQuestionOptions) =
         mutate()
     }
 
+    const isSubscribed = async (profile: StrapiRecord<ProfileData>): Promise<boolean> => {
```

Comment:
Any reason not to just grab this profile from the `useUser` hook directly rather than having it passed in?

Reply:
Valid! Fixed.